tests/components/jellyfin/__init__.py

```diff
@@ -1 +1,20 @@
 """Tests for the jellyfin integration."""
+import json
+
+from tests.common import load_fixture
+
+MOCK_AUTH_CONNECT_ADDRESS_SUCCESS = json.loads(
+    load_fixture("jellyfin/auth-connect-address.json")
+)
+
+MOCK_AUTH_CONNECT_ADDRESS_FAILURE = json.loads(
+    load_fixture("jellyfin/auth-connect-address-failure.json")
+)
+
+MOCK_AUTH_LOGIN_SUCCESS = json.loads(load_fixture("jellyfin/auth-login.json"))
+
+MOCK_AUTH_LOGIN_FAILURE = json.loads(load_fixture("jellyfin/auth-login-failure.json"))
+
+MOCK_GET_USER_SETTINGS_SUCCESS = json.loads(
+    load_fixture("jellyfin/get-user-settings.json")
+)
```

Comment:
`load_fixture` is doing I/O here.  Should these happen in the executor?

I'm not sure it actually matters so much in the tests though

Reply:
is there a better place in your opinion to let the io happen?

Reply:
I made the loading happen during actual mocking or tests vs always part of init